rwengine/include/script/ScriptFunctions.hpp

```diff
@@ -13,6 +13,8 @@
  * among many script modules and opcodes.
  */
 namespace script {
+const ScriptVec3 kSpawnOffset = ScriptVec3(0.f, 0.f, 1.f);
```

Comment:
I feel this commit should be in another PR if possible.

Please attempt to make this change a straight refactor.
It will be hard to review it otherwise.